### PR TITLE
Ltd 3242 Fix BEIS Nuclear Assessment Workflow

### PR DESCRIPTION
**Pull request by antroy-madetech**

### CODE REVIEW

File: caseworker/advice/views.py [hunk 1]
```diff
@@ -664,10 +670,10 @@ def get_context_data(self, **kwargs):
 
     def form_valid(self, form):
         data = {**form.cleaned_data}
-        selected_good_ids = data.pop("goods", [])
+        self.selected_good_ids = data.pop("goods", [])
```
Comment (saruniitr) [inline caseworker/advice/views.py:673]: I think this doesn't need to be in `self` now.

File: caseworker/advice/views.py [hunk 2]
```diff
@@ -623,7 +624,12 @@ class BEISProductAssessmentView(AdviceView, BEISNuclearMixin, BEISAssessmentBase
     form_class = BEISTriggerListAssessmentForm
 
     def get_success_url(self):
-        return reverse("cases:advice_view", kwargs=self.kwargs)
+        if self.case:
+            del self.case
```
Comment (saruniitr) [inline caseworker/advice/views.py:628]: Please add a comment about why we need to do this

File: ui_tests/caseworker/conftest.py [hunk 3]
```diff
@@ -447,7 +447,7 @@ def my_case_not_in_queue(driver, context):  # noqa
 @given("a queue has been created")  # noqa
 def create_queue(context, api_test_client):  # noqa
     api_test_client.queues.add_queue(f"queue {utils.get_formatted_date_time_y_m_d_h_s()}")
-    context.queue_id = api_test_client.context["queue_id"]
+    context.QUEUE_ID = api_test_client.context["queue_id"]
```
Comment (saruniitr) [inline ui_tests/caseworker/conftest.py:450]: any particular reason for this change?

Comment (antroy-madetech) [reply ui_tests/caseworker/conftest.py:450]: I think the IDE was being "smart". I'll roll these back

File: caseworker/advice/views.py (hunk 1 above)
Comment (antroy-madetech) [reply caseworker/advice/views.py:673]: Done.

File: caseworker/advice/views.py (hunk 2 above)
Comment (antroy-madetech) [reply caseworker/advice/views.py:628]: Done

**Pull request merged**